sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/ChangeFeedEncryptionProcessorBuilder.java

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.encryption;
+
+import com.azure.cosmos.ChangeFeedProcessor;
+import com.azure.cosmos.ChangeFeedProcessorBuilder;
+import com.azure.cosmos.CosmosAsyncContainer;
+import com.azure.cosmos.models.ChangeFeedProcessorOptions;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import reactor.core.scheduler.Schedulers;
+
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.stream.Collectors;
+
+/**
+ * Helper class to build a encryption supported {@link ChangeFeedProcessor} instance.
+ *
+ */
+public class ChangeFeedEncryptionProcessorBuilder {
```

Comment:
Beta annotations?

Reply:
@kushagraThapar  we decided not to add beta to all the classes and methods (which is GA in core project) of encryption as we have not done GA.
So I assume it will be applicable to this class as well. Please share the thoughts

Reply:
@milismsft  - We don't need Beta annotations for this one - since the whole project is in Beta preview only.
Once we decide to GA, then we can add Beta annotations if we need to, but for now, we don't need those. 
To be clear - all the new classes in azure-cosmos project related to encryption should have `@Beta` annotation. 
All new classes in azure-cosmos-encryption project need not to have any `@Beta` annotation.

Reply:
Thanks @kushagraThapar  for confirming